lib/runner.js

```diff
@@ -461,6 +461,29 @@ internals.executeTests = function (experiment, state, skip, callback) {
 };
 
 
+internals.experimentHasTests = function (experiment, state) {
+
+    if (experiment.experiments.length) {
+        return true;
+    }
+
+    const tests = experiment.tests.filter((test) => {
+
+        if ((state.filters.ids.length && state.filters.ids.indexOf(test.id) === -1) ||
+            (state.filters.grep && !state.filters.grep.test(test.title))) {
+
+            return;
+        }
+
+        if (!test.options.skip && test.fn) {
+            return test;
+        }
+    });
+
+    return tests.length;
```

Comment:
You should probably convert this to a Boolean before returning. "has tests" implies to me that it would be a Boolean.